SmartStorePerformance/ios/userstore.json

```diff
@@ -0,0 +1,104 @@
+{  "soups": [
```

Comment:
👍 

I guess we should change the SDK to get the configs file from a shared location so that you don't need two copies of the file (one for iOS and one for Android).
We did that for hybrid apps (https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/2484) but not for RN apps.